packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -275,6 +268,10 @@ class WebpackCLI implements IWebpackCLI {
       });
     };
 
+    // yarn uses 'add' command, rest npm and pnpm both use 'install'
+    const commandArguments = [packageManager === "yarn" ? "add" : "install", "-D", packageName];
```

Comment:
We can re-use the argument array if we keep it separate instead of concatenating immediately into a single string.